packages/hub/node-tests/routes/card-space-test.ts

```diff
@@ -341,3 +341,221 @@ describe('POST /api/card-spaces', function () {
+  });
+
+  it('updates the resource', async function () {
+    let dbManager = await server.container.lookup('database-manager');
+    let db = await dbManager.getClient();
+    await db.query(
+      'INSERT INTO card_spaces(id, url, profile_name, profile_description, profile_category, profile_button_text, profile_image_url, profile_cover_image_url, owner_address) VALUES($1, $2, $3, $4, $5, $6, $7, $8, $9)',
+      [
+        'AB70B8D5-95F5-4C20-997C-4DB9013B347C',
+        'satoshi.card.space',
+        'Satoshi Nakamoto',
+        "Satoshi's place",
+        'entertainment',
+        'Visit this Space',
+        'https://test.com/profile.jpg',
+        'https://test.com/cover.jpg',
+        '0x2f58630CA445Ab1a6DE2Bb9892AA2e1d60876C13',
+      ]
+    );
+
+    let payload = {
+      data: {
+        type: 'card-spaces',
+        attributes: {
+          url: 'satoshi.card.space',
+          'profile-name': 'Satoshi Nakamoto',
+          'profile-description': "Satoshi's place",
+          'profile-category': 'entertainment',
+          'profile-image-url': 'https://test.com/profile.jpg',
+          'profile-cover-image-url': 'https://test.com/cover.jpg',
+          'profile-button-text': 'Visit this Space',
+          'bio-title': 'Innovator',
+          'bio-description': "I'm a wealthy industrialist and philanthropist, and a bicyclist.",
+          links: [{ title: 'Link1', url: 'https://test.com' }],
+          'donation-title': 'The Human Fund',
+          'donation-description': 'A donation will be made in your name to the Human Fund',
+        },
+      },
+    };
+
+    await request
+      .put('/api/card-spaces/AB70B8D5-95F5-4C20-997C-4DB9013B347C')
+      .send(payload)
+      .set('Authorization', 'Bearer: abc123--def456--ghi789')
+      .set('Accept', 'application/vnd.api+json')
+      .set('Content-Type', 'application/vnd.api+json')
+      .expect(200)
+      .expect({
+        data: {
+          type: 'card-spaces',
+          id: 'ab70b8d5-95f5-4c20-997c-4db9013b347c',
+          attributes: {
+            url: 'satoshi.card.space',
+            did: 'did:cardstack:1csnaSutV4uMuyyJZcJ7ktsTwdec10adda76d48c7',
+            'profile-name': 'Satoshi Nakamoto',
+            'profile-description': "Satoshi's place",
+            'profile-category': 'entertainment',
+            'profile-image-url': 'https://test.com/profile.jpg',
+            'profile-cover-image-url': 'https://test.com/cover.jpg',
+            'profile-button-text': 'Visit this Space',
+            'owner-address': '0x2f58630CA445Ab1a6DE2Bb9892AA2e1d60876C13',
+            'bio-title': 'Innovator',
+            'bio-description': "I'm a wealthy industrialist and philanthropist, and a bicyclist.",
+            links: [{ title: 'Link1', url: 'https://test.com' }],
+            'donation-title': 'The Human Fund',
+            'donation-description': 'A donation will be made in your name to the Human Fund',
+          },
```

Comment:
Would this work to highlight the differences? But maybe it’s overkill 🤔

```suggestion
          attributes: {
            ...payload.data.attributes,
            did: 'did:cardstack:1csnaSutV4uMuyyJZcJ7ktsTwdec10adda76d48c7',
            'owner-address': '0x2f58630CA445Ab1a6DE2Bb9892AA2e1d60876C13',
          },
```